src/main/java/org/fundacionjala/prog101CoffeeMaker/coffeeMaker/Implementation/Controller.java

```diff
@@ -54,31 +63,56 @@ public void entryOption(final String option) {
         }
     }
 
+    /**
+     * Method for remove or load the pot
+     */
+    public void movingPot(final boolean wantPutPot) {
```

Comment:
The name of this method should be in the infinitive form

Reply:
ok